packages/jest-repl/src/cli/index.ts

```diff
@@ -5,26 +5,27 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
 import path from 'path';
 
 import Runtime from 'jest-runtime';
 import yargs from 'yargs';
+// @ts-ignore: Wait for jest-validate to get migrated
 import {validateCLIOptions} from 'jest-validate';
 import {deprecationEntries} from 'jest-config';
 import {version as VERSION} from '../../package.json';
 import * as args from './args';
 
 const REPL_SCRIPT = path.resolve(__dirname, './repl.js');
 
-module.exports = function() {
+exports = function() {
   const argv = yargs.usage(args.usage).options(args.options).argv;
 
   validateCLIOptions(argv, {...args.options, deprecationEntries});
 
   argv._ = [REPL_SCRIPT];
 
+  //@ts-ignore Get feedback first in PR
```

Comment:
The compiler complains that argv has some keys missing. Not sure how to resolve this at the moment.

Reply:
Yeah, the argv typing is not really where it should be. this is fine for now 🙂